acceptance/1_harvest_borrow/1_harvest_borrow.md

```diff
@@ -0,0 +1,85 @@
+# Harvest Borrow
+
+Acceptance instructions for basic borrowing functionality on Harvest.
+
+## Kvtool Testnet Setup
+
+First, start a kvtool testnet that contains the latest kava-labs/kava master branch changes and the relevant feature genesis.json file.
+
+```bash
+# Copy feature genesis.json file to kvtool's kava genesis template
+cp acceptance/1_harvest_borrow/genesis.json config/templates/kava/master/initstate/.kvd/config/genesis.json
+
+# Generate a new kvtool configuration based off template files
+kvtool testnet gen-config kava binance deputy --kava.configTemplate master
+
+# Pull the latest version Kava docker image containing the feature changing
+cd ./full_configs/generated && docker-compose pull
+
+# Start testnet
+cd ../.. && kvtool testnet up
+```
+
+You can test the set up by querying:
+
+```bash
+docker exec -it generated_kavanode_1 kvcli q harvest params
+```
+
+The output of the query should include a `money_markets` section at the bottom of the result.
+
+## Chain State Setup
+
+In order for USDX to be deposited and KAVA borrowed from HARD protocol the chain must have a valid state. The state must include valid asset prices in the pricefeed, access to an account loaded with USDX, etc.
+
+First, recover the oracle account and set up the pricefeed state.
+
+```bash
+# Recover oracle account
+docker exec -it generated_kavanode_1 kvcli keys add oracle --recover
+# Enter mnemonic
+desert october mammal tuition illness album engine solid enjoy harvest symptom rely camera unable okay avocado actual oppose remember lady dove canal argue cave
+
+# Set asset prices in price feed
+docker exec -it generated_kavanode_1 kvcli tx pricefeed postprice usdx:usd 1 99999999999 --from oracle
```

Comment:
Set prices in the genesis file.

Reply:
Updated in https://github.com/Kava-Labs/kvtool/pull/17/commits/213ecb786d7cf968cbffd99b8a7877e37f7561ed